pages/ksphere/kommander/1.1.0-beta/clusters/creating-konvoy-cluster-on-prem/index.md

```diff
@@ -29,6 +29,12 @@ See the [Install on-premise](/ksphere/konvoy/latest/install/install-onprem/) gui
 
 ![Add Cluster Options](/ksphere/kommander/1.1.0-beta/img/add-cluster.png)
 
-From the **Add Cluster** menu, select the **Upload YAML to Create a Cluster** option and select the **On Premise Cloud Provider** from the **Select Cloud Provider** field.
+In Kommander UI, choose the workspace where the on premise infrastructure provider was created in the header drop-down. From the workspace dashboard, select **Actions** > **Add Cluster**, and from the **Add Cluster** menu, select the **Upload YAML to Create a Cluster** option. Select your on premise infrastructure provider from the **Select Infrastructure Provider** select field.
```

Comment:
In the other PR, we usually write "On Premise".

Reply:
never on premise, only on premises.